src/Yaml/Parser.elm

```diff
@@ -581,7 +579,7 @@ recordInlineString =
     P.succeed ()
         |. P.chompWhile (U.neither U.isComma U.isRecordEnd)
         |> P.getChompedString
-        |> P.map Ast.fromString
+        |> P.map (Ast.fromString << String.trim)
```

Comment:
I don't really understand why you're doing this? I don't see how this addresses the issue?